packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -1141,7 +1141,7 @@ class _MenuItemButtonState extends State<MenuItemButton> {
         trailingIcon: widget.trailingIcon,
         hasSubmenu: false,
         overflowAxis: _anchor?._orientation ?? widget.overflowAxis,
-        child: widget.child!,
+        child: widget.child ?? const SizedBox.shrink(),
```

Comment:
Rather than just a one-line change here, would you be able to update [`_MenuItemLabel`](https://github.com/zeqinjie/flutter/blob/d895475c10af2201f938ac7a663e0425f2c18bba/packages/flutter/lib/src/material/menu_anchor.dart#L2972-L2985) so it has a nullable `child` property? Currently it's still documented as being required:

```dart
const _MenuItemLabel({
  required this.hasSubmenu,
  this.showDecoration = true,
  this.leadingIcon,
  this.trailingIcon,
  this.shortcut,
  this.semanticsLabel,
  this.overflowAxis = Axis.vertical,
  required this.child,
});

/// The required label child widget.
final Widget child;
```

Reply:
@nate-thegrate Thank you for your review，i have updated~